backend/migrations/2023-07-12-143111_user_data/up.sql

```diff
@@ -0,0 +1,33 @@
+-- Your SQL goes here
+CREATE TYPE salutation AS ENUM (
+    'Ms',
+    'Mrs',
```

Comment:
We should add a neutral  salutation for the third legal gender.
This is commonly done using just the persons name (E.g. "Dear Thorben Staufer, ..." instead of "Dear Mr Staufer").
Another option would be to address everyone only using their first name ("Dear Thorben, ...") regradless of their gender.